scripts/install/SetupAssignedTestCenterSyncUser.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 Open Assessment Technologies SA
+ */
+
+namespace oat\taoSync\scripts\install;
+
+use oat\generis\model\OntologyAwareTrait;
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\event\MetadataModified;
+use oat\tao\model\TaoOntology;
+use oat\taoSync\model\listener\ListenerService;
+use oat\taoSync\model\SyncService;
+use oat\taoSync\model\ui\SyncUserFormFactory;
+use oat\taoTestCenter\model\gui\form\TreeFormFactory;
+use oat\taoTestCenter\model\TestCenterService;
+
+class SetupAssignedTestCenterSyncUser extends InstallAction
+{
+    use OntologyAwareTrait;
+
+    /**
+     * Register the form for SyncUserAssigned in testcenter edit
+     *
+     * @param $params
+     * @return \common_report_Report
+     * @throws \common_Exception
+     */
+    public function __invoke($params)
+    {
+        try {
+            $this->registerTestCenterSyncUserProperty();
+            $this->registerTestCenterSyncUserForm();
+            $this->registerTestCenterOrgIdUpdatedEvent();
+            return \common_report_Report::createSuccess('SyncUserAssigned property for testcenter has been successfully set.');
+        } catch (\Exception $e) {
+            return \common_report_Report::createFailure('SyncUserAssigned property for testcenter has failed with message "' . $e->getMessage() . '".');
+        }
+    }
+
+    /**
+     * Register the property for TestCenterSyncUserAssigned
+     */
+    protected function registerTestCenterSyncUserProperty()
+    {
+        $property = $this->getProperty(SyncService::ASSIGNED_SYNC_USER);
+
+        if ($property->exists()) {
+            return;
+        }
+
+        $property->setRange($this->getClass(TestCenterService::ROLE_TESTCENTER_MANAGER));
+        $property->setMultiple(true);
+        $property->setDomain($this->getClass(TaoOntology::CLASS_URI_TAO_USER));
+        $property->setLabel('Assigned sync user');
+        $property->setComment('Assign sync user to the test center');
+        $property->setPropertyValue(
+            $this->getProperty('http://www.w3.org/2000/01/rdf-schema#subPropertyOf'),
+            'http://www.w3.org/2000/01/rdf-schema#member'
```

Comment:
This 2 constants have been remove before because we're not used, but because you are starting to use them I think you should add them to generis in the `OntologyRdfs` class